### DISCUSSION TITLE
Goilerplate is now templUI - Update on UI Components for Templ

### DISCUSSION THREAD

**Discussion by axadrn** [a-h/templ#1014] [Show and tell]
Hi Templ community! 👋

Quick update: Goilerplate has evolved and is now templUI! 

## Why the new name?

The rebranding better reflects what we're building - a professional UI component library specifically designed for Go and Templ projects. While the name has changed, our mission remains the same: providing type-safe, beautiful components for the Templ ecosystem.

## What's new since last time?

- Fresh branding and new website: templui.io
- Improved documentation
- More components and examples
- Better Alpine.js integration
- Enhanced Tailwind customization

## Next Steps

We're continuing to build and improve templUI. The project is still a work in progress, but we're excited about the direction and the feedback we've received from the community.

## Links

- [GitHub: github.com/axzilla/templui](github.com/axzilla/templui)
- [Website: templui.io](templui.io)

Thank you all for your continued support and feedback! Looking forward to building more together. 🚀

**Comment by pharrisee** [comment]
I'm definitely going to be looking at this for an upcoming work project, it's looking really good.

**Reply by axadrn** [reply]
Hey thanks man! Please let me know if you miss something!
